src/main/java/io/vertx/openapi/contract/impl/OperationImpl.java

```diff
@@ -72,6 +71,15 @@ public OperationImpl(String absolutePath, String path, HttpMethod method, JsonOb
       unmodifiableList(operationModel.getJsonArray(KEY_TAGS, EMPTY_JSON_ARRAY).stream().map(Object::toString).collect(
         toList()));
 
+    this.securityRequirements =
+      operationModel.containsKey(KEY_SECURITY) ?
+        unmodifiableList(
+          operationModel.getJsonArray(KEY_SECURITY).stream()
+            .map(o -> (JsonObject) o)
+            .map(SecurityRequirementImpl::new)
+            .collect(toList())) :
+        null;
```

Comment:
In case no securityRequirements are defined, we could return the global securityRequirements couldn't we?